### PR TITLE
Automatic timestamp

### PR DESCRIPTION
**Pull request by ruthmd**

### CODE REVIEW

File: j2v/utils/helpers.py [hunk 1]
```diff
@@ -1,45 +1,59 @@
...
+def is_unix_timestamp(dim_name, dim_val):
+    """
+       Checks if a string represents a timestamp
+       :param dim_name:
+       :param dim_val:
+       :return: True only if string represents a timestamp
+       """
+    digits = len(str(dim_val))
+    possible_digits = {10, 13, 16}
+    if dim_val > 0 and "time" in dim_name.lower() and digits in possible_digits:
```
Comment (thoean) [inline j2v/utils/helpers.py]: if <some condition>:
  return true;
return false;

can usually be written as:
return <some condition>

File: j2v/utils/helpers.py [hunk 2]
```diff
@@ -1,45 +1,59 @@
...
     Checks if a string represents a timestamp
-    :param potential_ts:
+    :param dim_name:
+    :param dim_val:
     :return: True only if string represents a timestamp
     """
     try:
-        datetime.datetime.strptime(potential_ts, "%Y-%m-%dT%H:%M:%S.%fZ")
-        return True
+        if datetime.datetime.strptime(dim_val, "%Y-%m-%dT%H:%M:%S.%fZ"):
```
Comment (thoean) [inline j2v/utils/helpers.py]: Why not use [fromisoformat](https://docs.python.org/dev/library/datetime.html#datetime.datetime.fromisoformat), which is broader since it allows full dates etc. instead of just the full timestamp?

File: examples/session_info/session_info.view.lkml [hunk 3]
```diff
@@ -70,28 +70,28 @@ view: session {
   }
     
   dimension: java_runtime {
-    description: "Java Runtime"
+    description: "Java_runtime"
```
Comment (thoean) [inline examples/session_info/session_info.view.lkml]: The previous version looked nicer. Is that change intentional?

Comment (ruthmd) [reply examples/session_info/session_info.view.lkml]: No this wasn't intentional, I'll fix it.
